### ISSUE TITLE
In plaats van $value en $error kan je ook de hele $RESULTS array meegeven aan de showFormItem

### ISSUE THREAD

**Issue by JeroenHeemskerk** [TobzT/educom-webshop-basis#16]
https://github.com/TobzT/educom-webshop-basis/blob/2d7ecc4d37c5ea4a871429190f38fdda219d7aa6/contact.php#L29-L34

Je ziet hier steeds een pattroon dat de eerste parameter $key ook degene is die je in $RESULTS uitvraagt bij $value en $error. Je kunt ook heel $RESULTS meegeven en de functie zelf de value en de error eruit halen.

**Comment by JeroenHeemskerk** [comment]
Ikzelf zou $RESULTS iets als $data noemen, want het is geen constante.

**Comment by TobzT** [comment]
ja, maar dat maakt testen en het opbouwen van andere forms met deze functie lastiger en ingewikkelder.

**Comment by TobzT** [comment]
Ik ben het niet eens met deze keuze, want het maakt de aanpasbaarheid en algemene toepassing van de functie specifieker, en ik heb liever dat ik zelf in elke situatie de values en errors aanlever, want die zijn in sommige situaties niet toepasbaar.
